src/fsharp/TypedTreePickle.fs

```diff
@@ -2579,6 +2579,7 @@ and p_expr expr st =
     | Expr.TyChoose (a, b, c)            -> p_byte 12 st; p_tup3 p_tyar_specs p_expr p_dummy_range (a, b, c) st
     | Expr.Quote (ast, _, _, m, ty)         -> p_byte 13 st; p_tup3 p_expr p_dummy_range p_ty (ast, m, ty) st
     | Expr.WitnessArg (traitInfo, m) -> p_byte 14 st; p_trait traitInfo st; p_dummy_range m st
+    | Expr.DebugPoint (_, innerExpr) -> p_expr innerExpr st
```

Comment:
Ok, so we are not serializing `Expr.DebugPoint`. Does this mean that inline functions won't have this information when they are used?

Reply:
> Ok, so we are not serializing Expr.DebugPoint. Does this mean that inline functions won't have this information when they are used?

That's correct, inline data carries no source information at present, and hasn't done so previously.

I did experiments in allowing inline data to carry source, but the call stacks get badly messed up, because the calling stack frame is present but the location is the inline code, and it's really hard to work out what's going on.